boards/arm/nrf52840_pca10056/dts.fixup

```diff
@@ -5,3 +5,5 @@
 
 #define CONFIG_SOC_FLASH_NRF5_DEV_NAME	NRF_NRF52_FLASH_CONTROLLER_4001E000_LABEL
 #define FLASH_DEV_NAME			CONFIG_SOC_FLASH_NRF5_DEV_NAME
+
+#define CONFIG_OT_PLAT_RADIO_DEVICE_NAME	NORDIC_NRF52840_RADIO_40001000_IEE802154_DRV_LABEL
```

Comment:
replace tab with spaces, also, use _DEV_NAME to shorten length.

Reply:
Do we want to have specific define for OpenThread here, shouldn't it be more generic?

Reply:
@KaSroka What name do you prefer?
Maybe `IEEE_802_15_4_RADIO_DEV_NAME` ?

Reply:
Sounds good